packages/primitives/src/Link/Link.tsx

```diff
@@ -25,3 +27,23 @@ export const Link = styled('a')<LinkProps>(
   ...systemProps,
   sx,
 )
+
+export function addToPayload(event) {
```

Comment:
Can we rename this `addHrefToPayload` so that when we import it in other files it is a bit more descriptive of what is being added to the event payload?